docs/rootless.md

```diff
@@ -23,29 +23,40 @@ penguin:231072:65536
 
 ### Distribution-specific hint
 
+Using Ubuntu kernel is recommended.
+
 #### Debian (excluding Ubuntu)
 * `sudo sh -c "echo 1 > /proc/sys/kernel/unprivileged_userns_clone"` is required
+* [`sudo modprobe overlay permit_mounts_in_userns=1` is required to use `overlay2` storage driver (recommended)](https://salsa.debian.org/kernel-team/linux/blob/283390e7feb21b47779b48e0c8eb0cc409d2c815/debian/patches/debian/overlayfs-permit-mounts-in-userns.patch)
 
 #### Arch Linux
 * `sudo sh -c "echo 1 > /proc/sys/kernel/unprivileged_userns_clone"` is required
 
 #### openSUSE
 * `sudo modprobe ip_tables iptable_mangle iptable_nat iptable_filter` is required. (This is likely to be required on other distros as well)
 
-#### RHEL/CentOS 7
+#### RHEL/CentOS 7 (excluding RHEL/CentOS 8)
```

Comment:
This is a bit ambiguous - does this mean; "it's not supported on RHEL/CentOS 8" or "it's not needed on RHEL/CentOS 8" ?

Perhaps remove it from the heading, and add a "note" section  below it, something like;


```Markdown
#### RHEL/CentOS 7

> **Note**: rootless mode is currently not supported on RHEL/CentOS 8
```

Reply:
Updated. It works on RHEL 8.